src/components/ChecWysiwyg.vue

```diff
@@ -0,0 +1,571 @@
+<template>
+  <div class="wysiwyg" :class="classNames">
+    <EditorContent
+      :editor="editor"
+      v-bind="sharedInputProps"
+      v-on="nonInputListeners"
+      @input="handleInput"
+    />
+    <label
+      v-if="label"
+      class="wysiwyg__label"
+      :data-content="label"
+      :for="id"
+    >
+      {{ label }}
+      <span
+        v-if="required"
+        class="wysiwyg__required-text"
+      >
+        {{ $t('general.required') }}
+      </span>
+    </label>
+    <EditorMenuBar v-slot="{ commands, isActive }" :editor="editor">
+      <div class="wysiwyg__menubar">
+        <button
+          class="wysiwyg__menubar--button"
```

Comment:
```suggestion
          type="button"
          title="Bold"
          class="wysiwyg__menubar--button"
```

Can you apply this to all of the buttons? BYO translations

Reply:
Good call